packages/contracts/contracts/registry/DAORegistry.sol

```diff
@@ -66,6 +73,37 @@ contract DAORegistry is InterfaceBasedRegistry {
         emit DAORegistered(daoAddr, _creator, _name);
     }
 
+    /// @notice Checks if the name is either 0-9, a-z or a dash (-).
+    /// @param _name The name of the DAO.
+    /// @return `true` if the name is valid or `false` if at least one char is invalid.
+    /// @dev Aborts on the first invalid char found.
+    function _checkNameValidity(string calldata _name) internal pure returns (bool) {
+        bytes calldata nameBytes = bytes(_name);
+        uint256 nameLength = nameBytes.length;
+        for (uint256 i; i < nameLength; i++) {
```

Comment:
Isn't it possible to optimize the `i++` as unchecked? Or still better to keep it since this is external input?

Reply:
+1

Reply:
done